addons/web/static/src/webclient/switch_company_menu/switch_company_menu.js

```diff
@@ -13,7 +13,7 @@ class CompanySelector {
         this.companyService = companyService;
         this.selectedCompaniesIds = companyService.activeCompanyIds.slice();
 
-        this._debouncedApply = debounce(() => this._apply(), toggleDelay);
+        this._debouncedApply = debounce(async () => this._apply(), toggleDelay);
```

Comment:
I'm not sure, it this async necessary, in this code we don't "await" for this._apply to happen.

Reply:
hello @jpp-odoo i was just trying to fix the problem with the runbot test, and forgot it there, can you tell me what could be causing it if you have some idea
Thanks !

Reply:
To solve the "Some services could not be started: company. Missing dependencies: field" Error

You need to add the field service in the test beforeEach:
(And import the fieldService of course : `import { fieldService } from "@web/core/field_service";`)
`serviceRegistry.add("field", fieldService);`

This should resolve the issue on the runbot.

This is because, in the tests we only have the minimum set of services, and if you need a specific one, you should add it.

Don't forget to do a test for this new feature :-)

Thanks.

Reply:
Thanks !